common/persistence/sql/sqlplugin/history_replication_dlq_tasks.go

```diff
@@ -60,8 +60,6 @@ type (
 	HistoryReplicationDLQTask interface {
 		// InsertIntoReplicationDLQTasks puts the replication task into DLQ
 		InsertIntoReplicationDLQTasks(ctx context.Context, row []ReplicationDLQTasksRow) (sql.Result, error)
-		// SelectFromReplicationDLQTasks returns one or more rows from replication_tasks_dlq table
-		SelectFromReplicationDLQTasks(ctx context.Context, filter ReplicationDLQTasksFilter) ([]ReplicationDLQTasksRow, error)
```

Comment:
cc @yux0 
I removed this from sql plugin interface. There's no caller, even on persistence manager layer, for it.